### DISCUSSION TITLE
[PVP] Ability to Choose a Different Submarine for Each Team

### DISCUSSION THREAD

**Discussion by Gamerism123** [FakeFishGames/Barotrauma#6165] [Ideas and suggestions]
As of now PvP mode only allows you to choose one submarine for both teams to use. The ability to choose a different submarine for each team would introduce the concept of asymmetry to the combat, where each team must capitalize on the best features of their submarine and see which one should come out the victor. Among sub designers this also adds a competitive element - who can make the best PvP sub? Could a Humpback beat a Typhon? How would a Dugong fare against a Remora?

There is now some (albeit recent) precedent in single-player's pirate missions where you can face off against a variety of ships and for the most part it seems this was well-received. I think extending this over to PvP would certainly be beneficial to the game.

**Comment by Stallfighter** [comment]
You kinda can do that already via console commands (spawn sub/rotate sub) but thats quite bulky

**Reply by Gamerism123** [reply]
Yeah, while I can appreciate that there are some workarounds, they're quite hacky, and I think this is suggestion would make sense as a base-game feature.

**Comment by Viibrant** [comment]
I would like to say that this is still of interest - been toying with the idea of a competitive matchmaking system

**Comment by Regalis11** [comment]
This has now (finally) been implemented in our private repo (https://github.com/FakeFishGames/Barotrauma-development/pull/5284), coming up in a future update. :)

**Reply by Viibrant** [reply]
The tease!! Will this be publically accessible? (API)
